pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/bloomfilter/BloomFilterHandler.java

```diff
@@ -111,6 +111,11 @@ public void updateIndices(SegmentDirectory.Writer segmentWriter, IndexCreatorPro
     }
   }
 
+  @Override
+  public void postUpdateIndicesCleanup(SegmentDirectory.Writer segmentWriter)
```

Comment:
Make this method a default method in the factory class and only implement it in ForwardIndexHandler. This will avoid having to create an empty method in all indexHandlers?

Reply:
done